core/domain/activity_domain.py

```diff
@@ -28,17 +28,38 @@ class ActivityReference(object):
     """
 
     def __init__(self, activity_type, activity_id):
+        """Constructs an ActivityReference instance.
+
+        Args:
+          activity_type: str.
```

Comment:
Please follow exactly the indentation in the examples given in the [style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments).

Also, this needs to say what the args represent (though in this case it's trivial). Viz.:

```
    activity_type: str. The activity type.
    activity_id: str. The activity id.
```